runtime/near-runtime-fees/Cargo.toml

```diff
@@ -14,3 +14,4 @@ Fees applied to near runtime encapsulated in a separate crate. Might merge it la
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
+near-primitives = { path = "../../core/primitives" }
```

Comment:
I was under the impression that we deliberately avoided using most of our crates here as our dependencies mess brings in `near-crypto` and WASM contracts get into a broken state (or just gets too big?). @nearmax is this the case?

Reply:
Correct, we cannot do it here. Also, `near-runtime-fees` should be publishable to crates.io which means we can only have dependencies on the crates that are in crates.io

Reply:
@bowenwang1996 Please, add a comment to this Cargo.toml, so we don't forget about this in the future.

Reply:
@nearmax Do you think there is a way we can have a good test for this?

Reply:
https://github.com/nearprotocol/nearcore/issues/2457